src/transformState.js

```diff
@@ -4,8 +4,31 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  const arr = Object.values(actions);
```

Comment:
actions is an array already, why do you need this?

Reply:
it was encrypted code)))